src/shop/search/regionNotification/RegionNotification.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useMemo } from 'react';
+import { ScrollView } from 'react-native';
+
+import { Trans, Typography } from '../../../_components';
+import { getPassHolderRegions } from '../../../_utils';
+import { useGetMe } from '../../../profile/_queries/useGetMe';
+import { TSearchFilters } from '../../_models/searchFilters';
+import { useGetRewards } from '../../_queries/useGetRewards';
+import * as Styled from './style';
+
+type TProps = {
+  filters: TSearchFilters;
+  onPress: () => void;
+  search: string;
+  searchAmount: number;
+};
+
+export const RegionNotification: FC<TProps> = ({ onPress, filters, search, searchAmount }) => {
+  const { data: searchOutOfRegionResults } = useGetRewards({
+    enabled: !filters.includeAllCardSystems && search.length > 0,
+    filters: { ...filters, includeAllCardSystems: true },
+    freeText: search,
+  });
+  const { data: passHolder } = useGetMe();
+
+  const regions = getPassHolderRegions(passHolder);
+  const calculatedAmount = useMemo(
+    () =>
+      searchOutOfRegionResults?.pages?.[0]?.totalItems > 2 ? searchOutOfRegionResults?.pages?.[0]?.totalItems - searchAmount : 0,
+    [searchOutOfRegionResults, searchAmount],
+  );
+
+  if (calculatedAmount === 0) return null;
+  return (
+    <>
+      <ScrollView>
```

Comment:
Waarom zit dat in ne scrollview?

Reply:
(En als dat moet, waarom is die notificationContainer dan nog nodig? Kunt ge dan niet gewoon ne padding van 15 (lijkt me ook niet juist, is 16 niet de defaultSpacing?) op de contentContainer van de scrollview zetten?